config/datadog_cijioagents2.yaml

```
@@ -0,0 +1,49 @@
+providers:
+  aks:
+    enabled: true
+datadog:
+  clusterName: 'cijioagents2'
```

```suggestion
  clusterName: 'cijenkinsio-agents-2'
```

Ref. https://github.com/jenkins-infra/terraform-aws-sponsorship/blob/main/eks-cijenkinsio-agents-2.tf#L16

We use diminutives for helmfiles but we must keep full FQDNs